test/Infer/external-uses.opt

```diff
@@ -0,0 +1,14 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -reinfer-rhs -souper-infer-inst -souper-exhaustive-synthesis %s > %t 2>&1
```

Comment:
We should not call with "-souper-infer-inst" here. The argument "-souper-exhaustive-synthesis" has been separated from "-souper-infer-inst".